fli-iam/shanoir-ng
# [fli-iam#2193] Manage processed datasets as VIP execution input

# Changes
- Allow processed datasets as VIP execution input
- Solr indexed processed dataset are linked to studyId, but no acquisition/exam/center...
- Recursively display processed datasets in study tree
- Change converter parameter name in `datasets/carmin-data/path/{completePath}` endpoint to `converterId`
- Recursively get acquisition/examination from processed datasets when :
   - naming files on download
   - checking access rights on dataset
   - creating processing resources
- Processing must always be linked to a study
   - Checked on front and endpoints side
- Processing must always have input dataset(s) linked to said study
- Correct bug on execution not failing if status unreachable through VIP API

# How to test
- Deploy on qualif
- Use existing processed datasets as input of `test-shanoir` VIP pipeline
- Use produced processed datasets as input of `test-shanoir` VIP pipeline, etc
- Check tree display
- Check processed datasets download (naming, etc)
